temporal-sdk/src/main/java/io/temporal/internal/sync/DeterministicRunnerImpl.java

```diff
@@ -81,42 +81,38 @@
   private static final int ROOT_THREAD_PRIORITY = 0;
   private static final int CALLBACK_THREAD_PRIORITY = 10;
   private static final int WORKFLOW_THREAD_PRIORITY = 20000000;
-  private final Runnable rootRunnable;
-
-  private static class NamedRunnable {
-    private final String name;
-    private final Runnable runnable;
-
-    private NamedRunnable(String name, Runnable runnable) {
-      this.name = name;
-      this.runnable = runnable;
-    }
-  }
+  static final String WORKFLOW_ROOT_THREAD_NAME = "workflow-method";
```

Comment:
Is there any particular reason these fields all got shuffled around? Makes it a little hard to find the relevant changes in the diff, so I generally try to put pure style changes like this in their own PRs.

Reply:
No, just noticed that this class has weird format while debugging. I'll make sure to to make a separate PR next time.